static/src/stylesheets/module/experiments/_giraffe.scss

```diff
@@ -1,20 +1,34 @@
 .giraffe {
-    @include f-header;
     width: 100%;
-    display: block;
-    margin-bottom: 1em;
-    border-top: 1px dotted $neutral-4;
+    padding: $gs-baseline / 2 $gs-gutter / 4 $gs-baseline;
+    background: $multimedia-main-2;
+    box-sizing: border-box;
 }
-.giraffe--heading {
-    margin-bottom: 0px;
-    margin-top: 4px;
-    color: $news-support-2  ;
-    line-height: 28px;
+
+.giraffe__heading {
+    @include fs-header(5);
+    color: $neutral-1;
+    display: inline;
 }
+
 .button--giraffe__icon {
     float: right;
 }
-.giraffe--copy {
-    color: $neutral-2;
-    margin-bottom: $gs-baseline;
+
+.giraffe__copy {
+    @include fs-header(5);
+    color: #ffffff;
+    display: inline;
 }
+
+.button--primary {
```

Comment:
I think this class is used throughout the whole site, so you'd better change it to `button--giraffe` to avoid side-effects.